lib/ecstatic.js

```diff
@@ -31,6 +31,17 @@ var ecstatic = module.exports = function (dir, options) {
   opts.root = dir;
   if (defaultExt && /^\./.test(defaultExt)) defaultExt = defaultExt.replace(/^\./, '');
 
+  // Support hashes and .types files in mimeTypes @since 0.8
+  if (opts.mimeTypes) {
+    if (typeof opts.mimeTypes === 'string') {
+      //TODO: should handleError have any effect here?
```

Comment:
@jfhbrook what do you think about the handleError option? Should we not throw if it is true?